components/sidebar_right_menu/sidebar_right_menu.jsx

```diff
@@ -111,13 +110,10 @@ export default class SidebarRightMenu extends React.Component {
     }
 
     getStateFromStores = () => {
-        const tutorialStep = PreferenceStore.getInt(Preferences.TUTORIAL_STEP, UserStore.getCurrentId(), 999);
-
         return {
             currentUser: UserStore.getCurrentUser(),
```

Comment:
Probably have sense to move this 3 to properties using mattermost-redux.

Reply:
In general, I'd agree, but I'm hesitant to start refactoring the sidebar in this PR. Would you be amenable to deferring as such?

Reply:
Agree, Is not a big deal to defer it.